package.json

```diff
@@ -3,7 +3,15 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "prettier": {
+    "useTabs": true,
+    "singleQuote": true,
+    "trailingComma": "all",
+    "bracketSpacing": false,
+    "semi": false
+  },
   "scripts": {
+    "prettier": "prettier --write 'v1/**/*.mjs' '*.js'",
```

Comment:
add `'*.mjs'`

Reply:
I swore the top-level was `.js`. I'll fix that.